### PR TITLE
[PRM-528] - Changed WYO to use outstanding amounts instead of total's

### PR DESCRIPTION
**Pull request by LewisMitchell**

### CODE REVIEW

**Comment by LewisMitchell** [comment]
https://github.com/hmrc/penalties/pull/48
https://github.com/hmrc/penalties-acceptance-tests/pull/57

File: app/services/PenaltiesService.scala [hunk 1]
```diff
@@ -73,16 +73,16 @@ class PenaltiesService @Inject()(connector: PenaltiesConnector) {
       allLPPs => {
         val allAdditionalPoints = allLPPs.filter(_.`type` == PenaltyTypeEnum.Additional)
         val allFinancialPoints = allLPPs.filter(_.`type` == PenaltyTypeEnum.Financial)
-        val estimatedLPPs = allAdditionalPoints.map(_.financial.amountDue).foldRight(BigDecimal(0))(_ + _)
-        val crystallisedLPPs = allFinancialPoints.map(_.financial.amountDue).foldRight(BigDecimal(0))(_ + _)
```
Comment (ruthdas03) [inline app/services/PenaltiesService.scala:77]: Can the 'estimatedLPPs' and 'crystallisedLPPs' be renamed, looks like they are actually getting the outstandingAmountDue from Additional and Financial

**Review by ruthdas03 — COMMENTED** [review]
There's a minor observation with regarding to naming, else everything looks ok to me.

File: app/viewmodels/IndexPageHelper.scala [hunk 2]
```diff
@@ -196,44 +195,21 @@ class IndexPageHelper @Inject()(p: views.html.components.p,
...
     } else None
   }
 
-  private def returnEstimatedVATMessageIfMoreThanZero(vatInterest: BigDecimal, isEstimatedVAT: Boolean, msgKeyToApply: String)(implicit messages: Messages): Option[String]={
-   if(vatInterest > 0) {
-    (isEstimatedVAT,vatInterest.isWhole() ) match {
-      case(true,true) => Some(messages(s"$msgKeyToApply.estimated", vatInterest))
-      case(true,false) => Some(messages(s"$msgKeyToApply.estimated", "%,.2f".format(vatInterest)))
-      case(false,false) => Some(messages(s"$msgKeyToApply", "%,.2f".format(vatInterest)))
-      case(false,true) => Some(messages(s"$msgKeyToApply", vatInterest))
```
Comment (ruthdas03) [inline app/viewmodels/IndexPageHelper.scala:225]: Similar naming observations here 'vatInterest' and 'isEstimatedVAT' and the method name 'returnEstimatedMessageIfInterestMoreThanZero' - which is being used as a generic one to display values and estimated values.

File: app/services/PenaltiesService.scala (hunk 1 above)
Comment (LewisMitchell) [reply app/services/PenaltiesService.scala:77]: They are getting the values from the same place but for different types of penalties - the estimated LPP's comes from the additional penalties that the user may have whereas the crystallised LPP's comes from all the LPP's for that period and have been finalised.

File: app/viewmodels/IndexPageHelper.scala (hunk 2 above)
Comment (LewisMitchell) [reply app/viewmodels/IndexPageHelper.scala:225]: Sorted!

**Pull request merged**